UI/src/view/point_asset/servicePointLabel.js

```diff
@@ -0,0 +1,160 @@
+(function(root) {
+
+  root.ServicePointLabel = function(groupingDistance) {
+    SignsLabel.call(this, this.MIN_DISTANCE);
+    var me = this;
+
+    this.MIN_DISTANCE = groupingDistance;
+
+    me.getSignType = function (sign) {
+      return sign.type;
+    };
+
+    var getSignExtensionType = function (sign) {
+      return sign.typeExtension;
+    };
+
+    me.getPropertiesConfiguration = function () {
+      return [
+        {signValue: [15], image: 'images/service_points/parkingGarage.png'},
+        {signValue: [12], image: 'images/service_points/parking.png'},
+        {signValue: [16], image: 'images/service_points/busStation.png'},
+        {signValue: [8], image: 'images/service_points/airport.png'},
+        {signValue: [9], image: 'images/service_points/ferry.png'},
+        {signValue: [10], image: 'images/service_points/taxiStation.png', height: 30},
+        {signValue: [6], image: 'images/service_points/picnicSite.png'},
+        {signValue: [4], image: 'images/service_points/customsControl.png'},
+        {signValue: [5], image: 'images/service_points/borderCrossingPoint.png', validation: validateText, height: 25},
+        {signValue: [13], image: 'images/service_points/loadingTerminalForCars.png', validation: validateText, height: 25},
+        {signValue: [14], image: 'images/service_points/parkingAreaBusesAndTrucks.png', validation: validateText, height: 25},
+        {signValue: [17], image: 'images/service_points/chargingPointElectricCars.png', validation: validateText, height: 25},
+        {signValue: [11], typeExtension: 5, image: 'images/service_points/railwayStation2.png'},
+        {signValue: [11], typeExtension: 6, image: 'images/service_points/railwayStation.png'},
+        {signValue: [11], typeExtension: 7, image: 'images/service_points/subwayStation.png'}
+      ];
+    };
+
+    me.getLabelProperty = function (sign, counter) {
+
+      var labelProperty = _.find(me.getPropertiesConfiguration(), function(properties) {
+        var includesSign = _.includes(properties.signValue, me.getSignType(sign));
+        if(properties.typeExtension)
+          return includesSign && properties.typeExtension === getSignExtensionType(sign);
+        else
+          return includesSign;
+      });
```

Comment:
instead of override all methods (findImage, getTextOffsetX ...), could be created a method only to return this
me.output = function(includesSign , properties) {
if(properties && properties.typeExtension)
          return includesSign && properties.typeExtension === getSignExtensionType(sign);
        else
          return includesSign;
}